dev_guide/deployments.adoc

```diff
@@ -286,11 +286,14 @@ deployment is created using the new tag value for the `helloworld` container.
 
 == Strategies
 
-A deployment configuration declares a strategy which is responsible for
-executing the deployment process. Each application has different requirements
-for availability (and other considerations) during deployments. OpenShift
-provides out-of-the-box strategies to support a variety of deployment
-scenarios.
+A deployment strategy determines the deployment process, and is defined by the
+deployment configuration. Each application has different requirements for
+availability (and other considerations) during deployments. OpenShift provides
+out-of-the-box strategies to support a variety of deployment scenarios.
+
+A deployment strategy adopts link:../dev_guide/application_health.html[readiness
```

Comment:
Hmm I changed it to 'adopts' because it didn't feel right using 'uses' with 'used' at the end of the same sentence. Changed it back though. I can see both arguments.